tests/streaming/test_resolver.py

```diff
@@ -35,7 +35,7 @@ def test_src_resolver_s3_connections(monkeypatch):
 
     client_cls_mock = mock.MagicMock()
     client_cls_mock.return_value = client_mock
-    resolver.LightningClient = client_cls_mock
```

Comment:
I think technically this was a bug. Mocking things on the global modules without using monkeypatching could result in hard to debug test interactions. This might be done in more places of the test suite, so that could give problems in the future.

Reply:
True. This should have been with monkeypatch